src/core/bpf/meson.build

```diff
@@ -13,6 +13,14 @@ clang_flags = [
         '-c',
```

Comment:
Maybe, better to rename `clang_flags` -> `bpf_clang_flags` for consistency with `bpf_gcc_flags`?

Reply:
Changed as requested.